gulp/build-tasks/modules.js

```diff
@@ -5,7 +5,7 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 module.exports = function (gulp, plugins, config, gulpWebpack, webpack, path) {
   return function () {
     //init is a global component the scripts inside of this are loaded on every page. For example - loader, license
-    let modules = ['loader', 'slider', 'autocomplete', 'pagination', 'data', 'misc', 'social-feed'];
+    let modules = ['loader', 'slider', 'autocomplete', 'pagination', 'data', 'misc', 'social-feed', 'jquery.status', 'jquery.history'];
```

Comment:
I think it would be better to have generic names instead of putting jquery in front like for examle jquery.status can be status because if we change the functionality from jquery to just js or Angular or something else.